core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.spec.ts

```diff
@@ -401,6 +414,24 @@ describe('Translation Suggestion Review Modal Component', function() {
         expect(component.startedEditing).toBe(false);
       });
 
+    it('should toggle the expansion state for the content area', () => {
```

Comment:
Split the into 2 separate tests and also mention when it should happen

Reply:
Done